vtex/address-form
# Update Ecuador address rules

#### What is the purpose of this pull request?

Update Ecuador address rules for missing localities, as described in the ticket [#200982](https://vtexhelp.zendesk.com/agent/tickets/200982) (Zendesk).

#### How should this be manually tested?

You can use the cart below to check the address-form both via Shipping Preview or Omnishipping for the updated regions.

- https://ecuador--vtexgame1.myvtex.com/checkout/cart/add?sku=312&qty=1&seller=1&redirect=true&sc=2

#### Types of changes
